src/html/html-script-tag.ts

```diff
@@ -22,6 +22,14 @@ import {Document, Import, ScannedImport, Severity, Warning} from '../model/model
  */
 export class ScriptTagImport extends Import { type: 'html-script'; }
 
+/**
+ * A synthetic import that provides the document containing the script tag to
+ * the javascript document defined/referenced by the script tag.
```

Comment:
javascript -> JavaScript and below

Reply:
Add something like "so that features in an HTML file and visible to scripts imported from that HTML files..."